packages/api/amplify_api/example/integration_test/graphql_tests.dart

```diff
@@ -44,7 +44,7 @@ void main() {
       final req = GraphQLRequest<String>(
           document: graphQLDocument, variables: <String, String>{'id': id});
       final response = await Amplify.API.mutate(request: req).response;
-      if (response.errors.isNotEmpty) {
+      if (response.errors != null) {
```

Comment:
The [spec](https://spec.graphql.org/June2018/#sec-Errors) does say that this should either be `null` or non-empty, but we should still probably check if it's empty since we don't want to report an empty array as having error'd and AppSync doesn't always conform perfectly to the spec if I remember correctly.

```suggestion
      if (response.errors != null && response.errors!.isNotEmpty) {
```